pkg/server/server.go

```diff
@@ -413,7 +400,12 @@ func (s *ProxyServer) removeFrontendsForStream(streamUID string) []*ProxyClientC
 			if frontend.frontend == nil {
 				continue
 			}
-			if frontend.frontend.streamUID == streamUID {
+			grpcFe, ok := frontend.frontend.(*grpcFrontend)
+			if !ok {
+				klog.V(5).ErrorS(nil, "Frontend is not a grpc Frontend", "connectID", frontend.connectID)
```

Comment:
This seems like noise. Prior to this change we used the continue in the previous if statement for http-connect. We still need to continue for http-connect but now http-connect is a frontend. So it needs to continue here but I don't think we need the klog.